indi-asi/asi_ccd.h

```diff
@@ -154,7 +154,7 @@ class ASICCD : public INDI::CCD
         uint8_t rememberVideoFormat = { 0 };
         ASI_IMG_TYPE currentVideoFormat;
 
-        INumber BlinkN[1];
+        INumber BlinkN[2];
```

Comment:
It's recommended to add enums to make this clearer.

enum
{
  BLINK_COUNT,
  BLINK_DURATION
};

then in code when you use BlinkN[BLINK_COUNT].value for example, it's clear to which element we are referring to.

Reply:
Done